org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF

```diff
@@ -32,3 +32,5 @@ Import-Package: org.apache.log4j;version="1.2.15",
  org.junit.runner;version="4.5.0",
  org.junit.runners;version="4.5.0"
 Automatic-Module-Name: org.eclipse.xtext.xbase.ui.tests
+Export-Package: org.eclipse.xtext.xbase.testlanguages.ui.tests;x-internal=true,
```

Comment:
this one is unrelated to the change, but caused by regeneration. leave it?

Reply:
who generates that?

Reply:
i dont have this change

Reply:
(also which workflow)

Reply:
maybe this one was not regenerated for a longer time.

Reply:
This is added when executing xtext-extras/org.eclipse.xtext.xbase.testlanguages/src/org/eclipse/xtext/xbase/testlanguages/GenerateXbaseTestLanguages.mwe2

Reproduced on master

Reply:
indeed 👍